app/experimenter/experiments/constants.py

```diff
@@ -151,9 +152,9 @@ class ExperimentConstants(object):
         for users that enroll into the experiment.
       </p>
       <p>
-        An <strong>{[1]}</strong> sends a Firefox addon which
+        An <strong>{[1]}</strong> sends a Firefox Add-On which
```

Comment:
I don't know if it matters, and we can always fix it some other day but the spelling seems to be "Add-ons" (not "Add-Ons").
<img width="834" alt="Screen Shot 2019-03-19 at 3 17 21 PM" src="https://user-images.githubusercontent.com/26739/54635303-38162c00-4a5a-11e9-919b-c06da0055834.png">
<img width="1068" alt="Screen Shot 2019-03-19 at 3 18 42 PM" src="https://user-images.githubusercontent.com/26739/54635348-4fedb000-4a5a-11e9-8877-98212f2a2ed9.png">

Reply:
Oh yeah, looks like `add-on` is actually valid: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons